.github/workflows/codeql-analysis.yml

```diff
@@ -21,25 +21,25 @@ defaults:
 
 jobs:
 
-  changes:
-    name: Check changes
+  conditional:
+    name: Check conditional workflows and jobs
     if: github.event_name != 'schedule' || github.repository == 'keycloak/keycloak'
     runs-on: ubuntu-latest
     outputs:
-      java: ${{ steps.changes.outputs.java }}
-      themes: ${{ steps.changes.outputs.themes }}
-      js-adapter: ${{ steps.changes.outputs.js-adapter }}
+      java: ${{ steps.conditional.outputs.codeql-java }}
+      themes: ${{ steps.conditional.outputs.codeql-themes }}
+      js-adapter: ${{ steps.conditional.outputs.codeql-js_adapter }}
     steps:
       - uses: actions/checkout@v3
 
-      - id: changes
-        uses: ./.github/actions/changed-files
+      - id: conditional
+        uses: ./.github/actions/conditional
```

Comment:
The full path here should be `./.github/actions/conditional/action.yml`. Does GitHub actions treat `actions.yml` as some kind of 'index' file to make this work?

Reply:
Nah, GH adds the action.yml part itself, you only need to refer to the dir where it exists.